CONTRIBUTION.md

```diff
@@ -4,9 +4,18 @@ Your contribution is welcome! Thank you for your interest in contributing to the
 
 ## Table of contents
 
-- [Developer Guide](#developer-guide)
-- [Code Contributions](#code-contributions)
-- [Bug Reports](#bug-reports)
+- [Contribute to the STACKIT CLI](#contribute-to-the-stackit-cli)
+  - [Table of contents](#table-of-contents)
```

Comment:
Nitpick, but is it common to have the table of contents in the table of contents? :sweat_smile: 
I think we can start with `Developer Guide` at the top

Reply:
Hm, I'm not sure why but the table of contents is automatically updated every time I save the file, and it includes _all_ headings. I've removed it and saved without formatting